lighthouse-core/report/html/report-styles.css

```diff
@@ -1105,6 +1123,16 @@
   --gauge-circle-size: var(--gauge-circle-size-big);
 }
 
+.lh-sticky-header--visible > .lh-gauge__wrapper--fail > .lh-gauge__svg-wrapper {
+  background-image: url("data:image/svg+xml,%3Csvg width='36' height='36' viewBox='0 0 36 36' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M17 26H20V23H17V26Z' fill='%23FF4E42'/%3E%3Cpath d='M17 20H20V11H17V20Z' fill='%23FF4E42'/%3E%3C/svg%3E");
+  height: 36px;
+  width:36px;
+}
+
+.lh-category-header .lh-gauge__wrapper--fail  .lh-gauge__svg-wrapper{
+  background-image: url("data:image/svg+xml,%3Csvg width='120' height='120' viewBox='0 0 120 120' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M56 80H64V72.0001H56V80Z' fill='%23FF4E42'/%3E%3Cpath d='M56 64H64V41H56V64Z' fill='%23FF4E42'/%3E%3C/svg%3E%0A");
```

Comment:
rather than modifying the SVG itself, let's try to use `background-size` to scale it up

maybe we can start with...

```css
background-size: calc(var(--gauge-circle-size-big) * 0.6);
```